### DISCUSSION TITLE
Using Quest finger-tracking data in a Unreal Engine 5 PC app

### DISCUSSION THREAD

**Discussion by progmars** [alvr-org/ALVR#1899] [General]
Hi.

I'm interested in integrating finger tracking in my Unreal Engine app, which would be running on a PC and not on Quest.

From the source code, I found that ALVR should expose Quest finger-tracking data to SteamVR if "Enable skeleton" is turned on. 

Unreal Engine is currently using OpenXR plugin, which seems to be working a bit differently from the old (now deprecated) OpenVR, but, in theory, it should still somehow receive finger data from ALVR, right? Has anybody tried that?

**Comment by zmerp** [comment]
From reports, UE OpenXR apps with ALVR are still broken. There is an issue with the target surface type
